test/support/action/accountsLogOut.spec.js

```diff
@@ -0,0 +1,23 @@
+import accountsLogout from "../../../src/support/action/accountsLogout";
+
+describe("accountLogin", () => {
+
+	const callOrder = [];
+
+	beforeEach(() => {
+		global.browser = {
+			deleteCookie: jest.fn(() => callOrder.push("deleteCookie")),
+			refresh: jest.fn(() => callOrder.push("refresh"))
+		};
+	});
+
+	it("should delete 2 Accounts cookies then refresh the page", ()=> {
+		accountsLogout();
+		expect(global.browser.deleteCookie).toHaveBeenCalledTimes(2);
+		expect(global.browser.deleteCookie).toHaveBeenNthCalledWith(1, "__nrpa_2.2");
```

Comment:
Is it necessary to use toHaveBeenNthCalledWith assertion if we have already asserted toHaveBeenCalledTimes above? Could these assertions instead just assert that global.browser.deleteCookie has been called with the expected cookie name?

Reply:
good point - I think although I could call toHaveBeenNthCalledWith twice, there could be further calls of it - I separately want to assert that it's been called exactly twice, at least I think was my reasoning!